django_scheduled_tasks/apps.py

```diff
@@ -0,0 +1,19 @@
+from django.apps import AppConfig
+from django.db.utils import OperationalError
+
+
+class GeneralConfig(AppConfig):
+    name = 'django_scheduled_tasks'
+
+    def ready(self):
+        from .models import ScheduledTask
```

Comment:
More of a question than a suggestion so feel free to resolve without changes: is there a reason the imports are put in the function instead of the top of the file with the other imports? e.g. performance reason?

Reply:
Yes - it's because apps.py is imported at a point where you end up in horrid import loops if you try to import models at the module-level, sadly!